dpp/src/errors/consensus/fee.rs

```diff
@@ -0,0 +1,7 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum FeeError {
+    #[error("Current credits balance {balance} is not enough to pay {fee} fee")]
+    BalanceIsNotEnoughError { balance: i64, fee: i64 },
```

Comment:
Why it is an i64 instead of u64? I don't think we allow subzero balances

Reply:
thanks for noticing, fixed